python-avd/pyavd/j2filters/generate_route_target.py

```diff
@@ -8,14 +8,14 @@
 
 def generate_route_target(esi_short: str | None) -> str | None:
     """
-    generate_route_target transforms 3 octets ESI like 0303:0202:0101 to route-target
+    generate_route_target transforms 3 octets ESI like 0303:0202:0101 to route-target.
 
     Parameters
```

Comment:
Maybe need to change this to Args to match google doc

Reply:
It is not the intend of this PR to change all docstrings to a common format. But indeed we need to do that at some point.